frontend/components/DuccKillerStatsList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import DuccStats, { IDucc } from "./DuccStats";
+import eventSource from "../data/EventSource";
+
+export default function DuccKillerStatsList() {
+  const [fetchedUsers, setFetchedUsers] = useState<IDucc[]>([
+    { user: "Loading..." },
+  ]);
+
+  useEffect(() => {
+    eventSource.onmessage = (e) => {
+      console.log(e);
+    };
+    eventSource.addEventListener("duccScore", (e: any) => {
+      const data = JSON.parse(e.data);
+      console.log(data.duccScores);
+      setFetchedUsers(data.duccScores.duccKillers);
+    });
+  }, []);
+
+  return (
+    <div>
+      <div className={"userlist-header"}>Top 10 Ducc Killers: </div>
+      <div className={"userlist-content"}>
+        {fetchedUsers.map((user) => (
+          <DuccStats key={user.user} user={user.user} duccs={user.duccs} />
+        ))}
+      </div>
+    </div>
+  );
+}
```

Comment:
DuccFriendStatsList.tsx and DuccKillerStatsList.tsx could be made into one component

Reply:
I was pulling my hair out when trying to figure it out even though it should be something simple. Would you mind giving it a try?